lua-docs/content/reference/player.yml

```diff
@@ -39,6 +39,9 @@ properties:
     - name: "Velocity"
       type: "Number3"
       description: "Player's velocity (speed + direction)."
+     - name: "Wave"
```

Comment:
Be careful with the indentation, the hyphen should be at the same column as the one on line 39.